api/cpp/docs/cmake_reference.md

```diff
@@ -5,14 +5,16 @@
 ## `slint_target_sources`
 
 ```
-slint_target_sources(<target> <files>.... [NAMESPACE namespace])
+slint_target_sources(<target> <files>.... [NAMESPACE namespace] [LIBRARY_PATHS name1=lib1 name2=lib2 ...])
 ```
 
 Use this function to tell cmake about the .slint files of your application, similar to the builtin cmake [target_sources](https://cmake.org/cmake/help/latest/command/target_sources.html) function.
 The function takes care of running the slint-compiler to convert `.slint` files to `.h` files in the build directory,
 and extend  the include directories of your target so that the generated file is found when including it in your application.
 
-The optional NAMESPACE argument will put the generated components in the given C++ namespace.
+The optional `NAMESPACE` argument will put the generated components in the given C++ namespace.
+
+Use the `LIBRARY_PATHS` argument to specify the name and paths to [component libraries](slint-reference:src/language/syntax/modules#component-libraries).
```

Comment:
maybe be a bit more explicit in the docs about the equal sign?

Reply:
Good point. I’ll add an example as well.